docs/template.md

```diff
@@ -0,0 +1,17 @@
+# Title
+
```

Comment:
What do you think about adding this information under the title?

Date: date of creation
Participants: list of people with names and email addresses
Scopes: list of affected scopes (core, platform, clients/javascript, tenderdash)
Projects: list of exact affected projects/packages (dashpay/dash, dashevo/grovedb, dashevo/platform/js-dash-sdk)

Reply:
Personally I like the Date and Participants part (not sure about including email). 
Scopes makes me think of a broader question of where we store ADRs aligned with multiple scopes - do they simply get placed in the highest relevant directory of our structure (assuming we do something like #1 vs a flat structure)?

Reply:
Also, does it make sense to include "Status" in the same place instead of its own complete section?

Reply:
I agree about scope, but projects are more about implementation although ADR is more abstract thing. Not sure what does it mean #1 but flat structure is required in case if we decide on several components, like asset lock dip where is we have Core and Platform in scope. I believe there will be more cases.

Reply:
> Not sure what does it mean #1 but flat structure is required in case if we decide on several components, like asset lock dip where is we have Core and Platform in scope. I believe there will be more cases.

I found that MADR team actually [recommends using directories](https://adr.github.io/madr/#usage-of-categories) for organizing. ADRs that relate to multiple things would just be located at the highest level. I would envision using the "Scope" to indicate what components they apply to.